tests/i18n/tests.py

```diff
@@ -1199,8 +1199,8 @@ def test_get_format_modules_lang(self):
     def test_get_format_modules_stability(self):
```

Comment:
It looks like this test was added to fix stability when passing `reverse=True`. See [this ticket](https://code.djangoproject.com/ticket/15129). (You can kind of see this since the calls to `reversed()` don't add anything to this test after this change.)

Reply:
Yes, we can remove this test.